unity/Assets/Scripts/DroneFPSAgentController.cs

```diff
@@ -61,22 +61,24 @@ public class DroneFPSAgentController : BaseFPSAgentController {
 #endif
         }
 
-        public override void RotateRight(ServerAction action) {
+        public void RotateRight(ServerAction action) {
```

Comment:
Is the drone supposed to be able to `Move{Ahead/Right/Left/Back}`? I thought @winthos mentioned yes to this a long time ago.

Removing such actions from base would mean they need to be added again here.

Reply:
Good point: I've added them in.